src/components/HorizontalCard.tsx

```diff
@@ -33,20 +13,18 @@ export interface IProps {
 
 const HorizontalCard: React.FC<IProps> = ({
   emoji,
-  title,
-  description,
   children,
   className,
   emojiSize,
 }) => (
-  <StyledCard className={className}>
-    <Emoji size={emojiSize} text={emoji} />
-    <Content>
-      <Title>{title}</Title>
-      <Description>{description}</Description>
+  <Flex borderRadius="base" className={className}>
+    <Emoji fontSize={emojiSize} text={emoji} />
+    <Text display="flex" flexGrow="0" flexShrink="1" flexBasis="75%" ml="8">
```

Comment:
Lets use `Flex` for this https://chakra-ui.com/docs/components/flex/usage
```suggestion
    <Flex grow="0" shrink="1" basis="75%" ml="8">
```